### ISSUE TITLE
[Visual Requirements – Make Code – tutorial] Ensure buttons have discernible text

### ISSUE THREAD

**Issue by Sheila111** [microsoft/pxt-microbit#3983]
**Describe the bug**
[Visual Requirements – Make Code – tutorial] Ensure buttons have discernible text

**Steps to reproduce the behavior**
1.Navigate to https://makecode.microbit.org/beta#
2.Open any tutorial eg. **Smiley Buttons**
3.Use Accessibility Insights tool and click **FastPass**

**Actual behavior**
[Visual Requirements – Make Code – tutorial] Ensure buttons have discernible text
![image](https://user-images.githubusercontent.com/59192069/114995256-37b61c80-9ed0-11eb-807c-29fa593406fe.png)

**Additional context**
1.OS: Windows(rs6)
2.makecode.microbit.org version: 3.1.75
3.Microsoft MakeCode version: 6.13.18

**Comment by Sheila111** [comment]
**Title**: WCAG 4.1.2: Ensures buttons have discernible text (.hintbutton)
**Tags**: Accessibility, WCAG 4.1.2, button-name
**Issue**: Ensures buttons have discernible text (button-name - https://accessibilityinsights.io/info-examples/web/button-name)
**Target application**: Microsoft MakeCode for micro:bit - https://makecode.microbit.org/beta#editor
**Element path**: .hintbutton
**Snippet**: <button class="ui button icon ui circular label blue hintbutton hidelightbox " tabindex="-1"><i class="icon lightbulb outline " aria-hidden="true" role="presentation"></i></button>
**How to fix**: 
Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element's default semantics were not overridden with role="none" or role="presentation"
  Element has no title attribute
**Environment**: Chrome version 89.0.4389.128

====
This accessibility issue was found using Accessibility Insights for Web 2.24.0 (axe-core 4.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
